netcon/includes/CFtp.h

```diff
@@ -75,6 +75,10 @@
 #define FTP_STATE_CANT_WRITE_FILE 15
 #define FTP_STATE_STARTUP 16
 
+#if WIN32
+typedef UINT_PTR SOCKET;
```

Comment:
How was the `SOCKET` type found before? Can't we import it from a system header?

Reply:
The system header is both `windows.h` as well as `winsock.h`, which is what I'm trying to avoid having on a header. It was found before by ordering of whoever added `CFtp.h`.

Also, this is actually not being used because `netgames` are not being compiled for Windows, but I guess they will be at some point?

Reply:
ok, they will definitely be compiled at some point